VTUL/dlp-access
# Adds support for 3d views. Updates combined 2d/3d view.

**Adds support for 3d views. Updates combined 2d/3d view.**
* * *


# Description/changes? (:star:)
* Decouples the 3d view from the combined view to support records that have a 3d model but no 2d tiles.
* Three use cases for Entomology collection should now work: 2d only / 3d only / both
* Brings 2d view (for all records) into the same layout as 3d and combined views


# How should this be tested?
Check that three use cases for the entomology project work correctly:

- 2d: https://3d-only.d2ysrrdhih4bgc.amplifyapp.com/archive/3b049t3w
- 3d: https://3d-only.d2ysrrdhih4bgc.amplifyapp.com/archive/r090w655
- both: https://3d-only.d2ysrrdhih4bgc.amplifyapp.com/archive/gs46cw3j

# Additional Notes:
- branch: `3d_only`
- **BIG BIG** thanks to @sushmadeegojuVT for their awesome work on these changes

# Interested parties
@otokama 
@goynejennifer
